logstreams/src/main/java/io/zeebe/distributedlog/restore/snapshot/impl/DefaultSnapshotRestoreRequest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.zeebe.distributedlog.restore.snapshot.impl;
+
+import io.zeebe.distributedlog.restore.snapshot.SnapshotRestoreRequest;
+
+public class DefaultSnapshotRestoreRequest implements SnapshotRestoreRequest {
```

Comment:
I'm not sure if we really need the interfaces fir SnapshotRestoreRequest and response

Reply:
Interfaces helps us to have another sbe serializable messages `SbeSnapshotRestoreRequest` and response. What would be another way of supporting sbe messages in this context?